assets/js/util.js

```diff
@@ -293,13 +305,62 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(!installationNotificationShown) {
+        apyRequestStartTime = Date.now();
+        apyRequestTimeout = setTimeout(function () {
+            displayUnobtrusiveWarning();
```

Comment:
Nope, I am clearing the `apyRequestTimeout` variable on `ajaxComplete`. Thus, if my AJAX request is completed before the INDIVIDUAL_THRESHOLD, the variable will be cleared and it will not execute.